src/index.html

```diff
@@ -17,11 +17,449 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
You are linking a SCSS file in your HTML. You should compile your SCSS to CSS and link the CSS file instead. Web browsers don't understand SCSS syntax.